<launch>
  <arg name="nodelet_manager" default="/kinect/kinect_nodelet_manager" />

  <arg name="standalone_nodelet" default="false" />

  <group unless="$(arg standalone_nodelet)">
    <node pkg="nodelet" type="nodelet" name="pc_thermal_synchronizer"
      args="load pandora_vision_hole/pc_thermal_synchronizer $(arg nodelet_manager)" output="screen" >
      <rosparam file="$(find pandora_vision_hole)/cfg/synchronizer_cfg/pc_thermal_synchronizer_params.yaml"
                command="load" />
    </node>
  </group>

  <group if="$(arg standalone_nodelet)">
    <node pkg="nodelet" type="nodelet" name="pc_thermal_synchronizer"
      args="standalone pandora_vision_hole/pc_thermal_synchronizer" output="screen" >
      <rosparam file="$(find pandora_vision_hole)/cfg/synchronizer_cfg/pc_thermal_synchronizer_params.yaml"
                command="load" />
    </node>
  </group>
</launch>
